config/locales/activerecord.yml

```diff
@@ -1,20 +1,68 @@
 en:
+  number:
+    format:
+      delimiter: ","
```

Comment:
Did this solve the number formatting? 🤔

Reply:
No, it doesn't. 😞

Reply:
Will remove this.

Reply:
In that case, I think it's better to remove, thanks 😄